virt/virtual_machines/vm_networking/virt-using-the-default-pod-network-with-virt.adoc

```diff
@@ -1,14 +1,12 @@
 :_content-type: ASSEMBLY
 [id="virt-using-the-default-pod-network-with-virt"]
-= Using the default pod network for virtual machines
+= Configuring the default pod network
 include::_attributes/common-attributes.adoc[]
 :context: virt-using-the-default-pod-network-with-virt
 
 toc::[]
 
-You can use the default pod network with {VirtProductName}. To do so, you must use the `masquerade` binding method. Do not use `masquerade` mode with non-default networks.
-
-xref:../../../virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc#attaching-to-multiple-networks[For secondary networks], use the `bridge` binding method.
+By default, {VirtProductName} is installed with a single, internal pod network. To use it, bind with `masquerade` mode. Do not use `masquerade` mode with non-default networks.
```

Comment:
First sentence is fine. Second sentence ("To use it...) needs to be rewritten. Suggestion: "You must configure the network interface of a virtual machine to use the `masquerade` binding mode."

About not using masquerade mode for external networks (not non-default), I'm not sure that's true. Look at this screenshot from the yaml in the Linux bridge assembly. Clearly both masquerade and bridge are used. In fact, I am wondering whether the procedures should be combined. 

![image](https://user-images.githubusercontent.com/31725355/166653548-685aa295-d15f-49c7-88ec-84d20961e26f.png)

Reply:
Done.